portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -113,6 +113,7 @@ private String convertToJsonUsingGson(List toConvert) {
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    System.out.println("doPost()");
```

Comment:
Why is this needed?

Reply:
I think I was planning on leaving it until I finished implemented all of blobstore to make sure it works, but I just removed it so it won't be merged to master.

Done!